databuilder/databuilder/task/neo4j_staleness_removal_task.py

```diff
@@ -182,19 +195,16 @@ def _batch_delete(self,
                 total_count = total_count + count
                 if count == 0:
                     break
-            LOGGER.info('Deleted %i stale data of %s', total_count, t)
+            LOGGER.info('Deleted %i stale data of %s', total_count, type)
 
     def _validate_staleness_pct(self,
                                 total_records: Iterable[Dict[str, Any]],
```

Comment:
can we just pass numbers here instead of this Iterable[Dict[str, Any]]. That will make this function simpler and I don't see why passing Iterable[Dict[str, Any]] was ever necessary.